src/types/AddContext/AddContext.ts

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Looks like the same class as 
https://github.com/adobe/genstudio-uix-sdk/blob/main/src/types/generationContext/GenerationContextService.ts 
Should we just rename?